src/renderer/aboutDialog/index.html

```diff
@@ -0,0 +1,57 @@
+<html>
+    <head>
+        <title>GPII Error</title>
+        <link rel="stylesheet" href="../../../node_modules/winstrap/dist/css/winstrap.min.css">
+        <link rel="stylesheet" href="../psp/css/button.css">
+        <link rel="stylesheet" href="../common/css/dialog.css">
+        <link rel="stylesheet" href="../common/css/titlebar.css">
+        <link rel="stylesheet" href="../common/css/heightObservable.css">
+        <!-- TODO extract to generic place -->
+        <link rel="stylesheet" href="../errorDialog/css/main.css">
+
+        <script type="text/javascript" src="../../../node_modules/infusion/src/lib/jquery/core/js/jquery.js"></script>
+        <script type="text/javascript" src="../../../node_modules/infusion/dist/infusion-all.js"></script>
+
+        <script type="text/javascript" src="../psp/js/widgets.js"></script>
+        <script type="text/javascript" src="../common/js/titlebar.js"></script>
+        <script type="text/javascript" src="js/aboutDialog.js"></script>
+        <script type="text/javascript" src="js/index.js"></script>
+
+        <script type="text/javascript" src="../common/js/heightObservable.js"></script>
```

Comment:
Remove the inclusion of this script, along with the styles above for the height observable. No need to have these as the about dialog is not resizable.